sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 4.2.0-beta.2 (Unreleased)
+## 4.2.0-beta.2 (2020-10-06)
```

Comment:
Despite the fact that all of these changes below are important, their limited cloud availability tells me that I shouldn't bring this up to the release highlights for this release.

Reply:
fair enough, we can add it to the next one